### ISSUE TITLE
Model comparer

### ISSUE THREAD

**Issue by AyushExel** [AyushExel/trolo#29]
It'd be nice to have a tool where you can select models and input source and it run outputs side-by-side depending on the models. 
Ideally, it run the models in parallel as separate processes for best performance idea.

**Comment by hardikdava** [comment]
@AyushExel Are you thinking of comparing only trolo models or external models as well?

**Comment by AyushExel** [comment]
Currently I was thinking of models supported in trolo. I've got plans to add many architectures like RTDETR etc. Its only a matter of compute

**Comment by hardikdava** [comment]
@AyushExel can you elaborate on spec details? I can design very minimal script to achieve this.

- Is it for only 1 image or selected images?
- Do we have detection results saved on disk?
- Does slider needed to compare results in high precision?
